tests/target/cargo-toml/true/basic/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+# comment before package
+[package]
+name = 3
+version = 1
+# comment 1  for arr1 line1
+# comment 1 for arr1 line 2
+arr1 = [1, 2, 3]
+# comment 2   for arr2
+arr2 = [
+    "11111111111111111111111111111111111111111111111111111111111111111111111111111111",
+    "1111111111111111111111111111111111111111111111111111111111111111111111111111111"
+]
+description = """
+a
+b
+haha"""
+
+# rustfmt-format_cargo_toml: true
```

Comment:
At first I thought that this comment was removed since I normally expect to find these comments at the top of the file. This should be fine as long as `format_cargo_toml=true` is set in the Config. When you have a chance, can you look into this and confirm that's what's happening?

Reply:
Yes, this reordering is expected, as the line below says.

Reply:
And the option is correctly set
<img width="761" alt="image" src="https://user-images.githubusercontent.com/37948597/202914895-efcdd739-a783-471a-8f20-b12bf7cb2127.png">

Reply:
Thank you for confirming!